utils/btrfs-progs/Makefile

```diff
@@ -53,6 +57,9 @@ CONFIGURE_ARGS += \
 	--disable-convert \
 	--disable-documentation \
 	--disable-python
```

Comment:
Missing ```\``` ?

Reply:
Not in this case. \ would make the if statement itself part of the configure args.

Reply:
Ahh, I guess that depends on how you write if statements as this works just fine
https://github.com/diizzyy/packages/blob/ffmpeg4/multimedia/ffmpeg4/Makefile#L195

Reply:
@diizzyy just so you know those if statements in Makefile you linked are possibly wrong. Kconfig defines two states of deactivated option `# CONFIG_* is not set` and `CONFIG_*=n` where second one is strictly disabled while first one is optionally disabled and can be reenabled without configuration restart. These two states are valid and should be expected although mconf does not produce second state up to my knowledge.

With this the syntax `$(if $(CONFIG_*),)` is evaluated as valid for even disabled option. The correct approach would be probably something like `$(if $(filter y,$(CONFIG_*)),)` and in that case I consider appending to variable to be a better option.

Reply:
No worries, I just used the curl package in main repo as template.

Reply:
Nobody says that packages in main repository are bug free. Just so you know I tested it and variable is really defined as `n` when `CONFIG_FOO=n` is used, which breaks approach in those packages. The solution could be to do config file post-processing to unify those two options. This is for sure bug in those packages or in build system. Well oversight bug but bug non the less.

This is not some theoretical thing just so you know. The `CONFIG_FOO=n` is used when you generate configuration from some subset using Kconfig. Even OpenWRT is using it in target kernel configuration.

The true is that this is outside of this PR. Can we merge this?

Reply:
Agree, merged. Thanks